qiskit/providers/ibmq/api/session.py

```diff
@@ -198,10 +198,7 @@ def request(  # type: ignore[override]
                     pass
 
             if self.access_token:
-                message = message.replace(self.access_token, '...')
-                # Replace the original message on the `RequestException`.
-                ex.args = (message,)
-                # Replace the original message on the chained exceptions.
+                # Modify the original message on the chained exceptions.
```

Comment:
I think you still need `message = message.replace(self.access_token, '...')` ? Since it's used in line 204.